libvast/src/command.cpp

```diff
@@ -222,6 +222,26 @@ void mantext(const command& cmd, std::ostream& os,
   os << '\n';
 }
 
+/// Converts a verbosity atom to its integer counterpart.
+unsigned to_level_int(caf::atom_value x) {
+  switch (caf::atom_uint(caf::to_lowercase(x))) {
+    default:
+      return VAST_LOG_LEVEL_QUIET;
+    case caf::atom_uint("quiet"):
+      return VAST_LOG_LEVEL_QUIET;
+    case caf::atom_uint("error"):
+      return VAST_LOG_LEVEL_ERROR;
+    case caf::atom_uint("warning"):
+      return VAST_LOG_LEVEL_WARNING;
+    case caf::atom_uint("info"):
+      return VAST_LOG_LEVEL_INFO;
+    case caf::atom_uint("debug"):
+      return VAST_LOG_LEVEL_DEBUG;
+    case caf::atom_uint("trace"):
+      return VAST_LOG_LEVEL_TRACE;
+  }
+}
```

Comment:
This seems awfully similar to the code below in `init_config(...)` in the same file. This should be deduplicated.

Reply:
The same code also exists in `vast/system/logger.cpp`.

Reply:
Good spot; deduplicated.